_includes/sidenav.html

```diff
@@ -50,11 +46,14 @@ <h1 class="u-visually-hidden">
         </button>
     </div>
     <div class="expandable_content">
+        <hr class="inset-divider" />
         <ul class="nav-secondary_list">
         {%- for href, id, caption in items %}
-            <li class="{{ 'nav-secondary_item' if loop.index == 1 else 'nav-secondary_subitem' }}">
+            {# TODO: logic around loop.index needs to be updated
+                     so a parent item can appear before first item. #}
+            <li class="nav-secondary_item{{ ' nav-secondary_item-parent' if loop.index == 1 else ' nav-secondary_item-child' }}">
```

Comment:
Yes, sounds good. The parent designation is needed to target the parent link only. All the items have `nav-secondary_item`, the browse pages will have `nav-secondary_item__parent` and the browse page's subpages will have `nav-secondary_item__child`.